test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,60 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+const path = 'https://httpbin.org';
+
+const query = {
+    name: 'Carlo',
+    age: '19',
+    city: 'New York'
+};
+
+describe('First Api Tests', () => {
+    //Returns Origin IP.
+    it('Consume GET Service', async () => {
+        const response = await axios.get(`${path}/ip`);
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data).to.have.property('origin');
```

Comment:
As suggestion: its a good idea to separate different actions, at least with spaces
For instance here, `arrange` to setup everything thats need it to the tests (variables definitions...)
`assert` to do all the tests
```suggestion
        const response = await axios.get(`${path}/ip`);
        
        expect(response.status).to.equal(StatusCodes.OK);
        expect(response.data).to.have.property('origin');
```